### ISSUE TITLE
[MAINT] Setting up CI with GitHub Actions

### ISSUE THREAD

**Issue by marsipu** [mne-tools/mne-qt-browser#2]
As suggested, I set up CI for the associated tests for the pg-backend.

Currently to run viz-tests for the pg-backend, just the parameter of the fixture `browse_backend` hast to be set to "pyqtgraph".

My first shot on this was to create `conftest.py` within this repository for pyqtgraph and to copy the test-files from mne to run them in the directory of this repo (I didn't find a way to pass parameters to pytest when it runs in mne-python to just run "pyqtgraph").
There might be a better solution. WDYT?

With this solution we could maybe temporarily remove "pyqtgraph" from `browse_backend` in MNE-Python in the current integration PR to make it pass the tests there and work on passing the tests from this repo here.

**Comment by agramfort** [comment]
Yes the testing time of this feature should be spent on the CIs of this repo. You use the mne testing data or maybe simply the very small raw file in the mne python repo or even use random data if you think it’s enough.

**Comment by marsipu** [comment]
Ok, yes the tests from `mne.viz.test_raw.py` use the small raw-file from `mne.io`.

**Comment by marsipu** [comment]
The CIs are still failing for numerous reasons depending on the operating system.
When I got time I will look into them.
